packages/@okta/vuepress-site/docs/guides/set-up-token-exchange/main/index.md

```diff
@@ -125,58 +125,65 @@ An [access policy](/docs/guides/configure-access-policy/main/) helps you secure
+* `response_type` is `code`, indicating that we are using the Authorization Code grant type.
+* `scope` is `openid`, which means that the `/token` endpoint returns an ID token. See the **Create Scopes** section of the [Create an authorization server](/docs/guides/customize-authz-server/main/#create-scopes) guide.
+* `redirect_uri` is the callback location where the user agent is directed to along with the code. This must match one of the **Sign-in redirect URIs** that you specified when you created your ntaive app.
+* `state` is an arbitrary alphanumeric string that the authorization server reproduces when redirecting the user agent back to the client. This is used to help prevent cross-site request forgery.
+* `code_challenge_method` is the hash method used to generate the challenge, which is always `S256`.
+* `code_challenge` is the code challenge used for PKCE.
+
+> **Note:** See the [OAuth 2.0 API reference](/docs/reference/api/oidc/#authorize) for more information on these parameters.
+
+If the user doesn't have an existing session, this request opens the Okta sign-in page. If they have an existing session, or after they authenticate, the user arrives at the specified `redirect_uri` along with an authorization `code`:
 
 **Response**
 
 ```bash
-https://${configuredRedirectUri}/?code=FQGFlDO-J1jXl....7-cfYJ0KtKB8&state=testState`
+https://${configuredRedirectUri}/?code=FQGFlDO-J1jXl....7-cfYJ0KtKB8&state=testState
 ```
+#### Exchange code for tokens request
 
-**Exchange code for tokens request**
+Use the following example to build the request to exchange the authorization code for tokens.
 
-```curl
-curl --location --request POST \
-  --url 'https://${yourOktaDomain}/oauth2/{authServerId}/v1/token' \
-  --header 'Accept: application/json' \
-  --header 'Content-Type: application/x-www-form-urlencoded' \
-  --data-urlencode 'grant_type=authorization_code' \
-  --data-urlencode 'redirect_uri=${configuredRedirectUri}' \
-  --data-urlencode 'code=FQGFlDO-J1j.....QvabuZ7-cfYJ0KtKB8' \
-  --data-urlencode 'code_verifier=xO5wgOEH5UA2XUdVQ88pM.....Rtc5ERKq1MeonMo8QLCSRYlDk' \
-  --data-urlencode 'client_id=${nativeAppClientId}'
+```bash
+  curl --location --request POST \
+    --url 'https://${yourOktaDomain}/oauth2/{authServerId}/v1/token' \
+    --header 'Accept: application/json' \
+    --header 'Content-Type: application/x-www-form-urlencoded' \
+    --data-urlencode 'grant_type=authorization_code' \
+    --data-urlencode 'redirect_uri=${configuredRedirectUri}' \
+    --data-urlencode 'code=FQGFlDO-J1j.....QvabuZ7-cfYJ0KtKB8' \
+    --data-urlencode 'code_verifier=xO5wgOEH5UA2XUdVQ88pM.....Rtc5ERKq1MeonMo8QLCSRYlDk' \
+    --data-urlencode 'client_id=${nativeAppClientId}' \
 ```
 
 **Response**
 
 ```json
-{
+  {
     "token_type": "Bearer",
     "expires_in": 3600,
     "access_token": "eyJraWQiOiJ6………X1Z4QA",
     "scope": "openid"
```

Comment:
nit: since you've requested the `openid` scope an `id_token` is going to be returned here